tests/dspy/test_dspy_autolog.py

```diff
@@ -19,16 +19,23 @@
 from mlflow.entities import SpanType
 from mlflow.entities.trace import Trace
 from mlflow.models.dependencies_schemas import DependenciesSchemasType, _clear_retriever_schema
+from mlflow.models.model import _MODEL_TRACKER
 from mlflow.tracing.constant import SpanAttributeKey
 from mlflow.tracking import MlflowClient
 
+from tests.dspy.test_save import CoT
 from tests.tracing.helper import get_traces, score_in_model_serving
 
 _DSPY_VERSION = Version(importlib.metadata.version("dspy"))
 
 _DSPY_UNDER_2_6 = _DSPY_VERSION < Version("2.6.0rc1")
 
 
+@pytest.fixture(autouse=True)
+def reset_model_tracker():
+    _MODEL_TRACKER.clear()
```

Comment:
Can we move this to `tests/conftest.py`?

Reply:
Ah it's already there :) Let me remove this